modules/core/src/main/scala/com/caesars/tracing/TracingUtils.scala

```diff
@@ -0,0 +1,55 @@
+package com.caesars.tracing
+
+import io.janstenpickle.trace4cats.inject.EntryPoint
+import io.janstenpickle.trace4cats.kernel.{SpanCompleter, SpanSampler}
+import io.janstenpickle.trace4cats.model.TraceProcess
+import io.janstenpickle.trace4cats.newrelic.{Endpoint, NewRelicSpanCompleter}
+import org.asynchttpclient.DefaultAsyncHttpClientConfig
+import org.http4s.asynchttpclient.client.AsyncHttpClient
+import org.http4s.client.Client
+import org.http4s.client.middleware.Logger
+import zio.*
+import zio.interop.catz.*
+import zio.interop.catz.implicits.rts
+
+object TracingUtils {
+  private val sampler: ULayer[Has[SpanSampler[Task]]] = ZLayer.succeed(SpanSampler.always[Task])
+  private val client: ULayer[Has[Client[Task]]] =
+    // Blaze client does not support proxy, se we use Async client implementation (same underlying impl as Sttp)
```

Comment:
> Blaze client

Is this a reference to `http4s`? Or STTP or another library has a `blaze` component?

Reply:
Yes, I believe this is a reference to http4s. If you take a look at the line directly below, an http4s Client is source, that's backed by the ning async http client instead of Blaze.

While my name is associated with the MR, I'm not responsible for any of the code here - you might want to follow up with others to be sure.

Reply:
Any idea who wrote it, Alex? If so, please share so I can `@` them. Or, to avoid blocking you, we could make an issue in this repo (if that's how we want to track tickets) and `@` those folks?